src/utils/numberValidations.ts

```diff
@@ -0,0 +1,8 @@
+export function isValidNumber(value: any): number {
```

Comment:
Ignore my former suggestion, you are right that `any` is actually ok here as it’s a method checking the input so implies any input should be passable. But the return type based on actual code is really `boolean`

Reply:
Yeah, I'm not sure why I added `number`, thanks for noticing 😂 😂